include/onnxruntime/core/graph/graph.h

```diff
@@ -1132,6 +1133,19 @@
     return domain_to_version_;
   }
 
+  // Data structure stores prepacked initializers in format of Tensor.
+  struct PrePackInitializers {
```

Comment:
>struct PrePackInitializers [](http://example.com/codeflow?start=2&length=26)

Generally, we want to obey the layering.
Graph deals with the graph structures and Tensors are usually separate. Can we put this and the related methods into graph_utils or some other place?